js/main.js

```diff
@@ -1,30 +1,99 @@
+    Title: $Title.value, // in the memory location of the object variable
     PhotoUrl: $photoUrl.value,
     Notes: $Notes.value,
-    NextEntryId: data.nextEntryId
+    NextEntryId: data.nextEntryId// also of the nextEntryId from data.js
   };
 
-  data.entries.unshift(formInput);
-  data.nextEntryId++;
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  data.entries.unshift(formInput);// the new object is being pushed into the array entries of the
+  data.nextEntryId++; // data object, and the data entryId is incremented
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');// updated image
   $form.reset();
-  // localStorage.setItem('Form-Input-Stringified', JSON.stringify(formInput));
+  // MOVES DATA ENTRY
+
+  while ($entriesSpace.firstChild) { // it avoids the over lapping of previous content in entries with new content
+    $entriesSpace.removeChild($entriesSpace.firstChild);// by removing the first child of $entriesSpace
+    // which is the elements that have been created
+  }
+  movesDataEntry();// it will now run this function when i click on the save btn in the form
+}
+// issue 2
+var $entriesSpace = document.querySelector('.entries-space');
+var $entriesBtn = document.querySelector('#entries');
+$entriesBtn.addEventListener('click', takesJournalEntry);
+
+function takesJournalEntry(indexValue) { // function for when the entries title is clicked
+// hides the form and show entries
+  backToEntries();// it calls the function that automatically switches the view swap from
+  // the form to the entries
+  if (!data.entries[indexValue]) { // logic gate to make sure theres data in the entries so it wont get an error when trying to switch and theres nothing
+    return;
+  }
+  var $divImage = document.createElement('DIV');// creates div element
+  $divImage.classList.add('style-for-image-div');// adds a class for styling purposes
+
+  var $img = document.createElement('IMG');// creates img element
+  $img.classList.add('width-for-image');
+  $img.setAttribute('src', data.entries[indexValue].PhotoUrl);// assigns src to the img tag
+  $img.setAttribute('alt', 'images');// assigns alt to the img tag
+
+  var $rightContainer = document.createElement('DIV');
+  $rightContainer.classList.add('size-of-right-container');// creates a div for the text
+  var $titleH3 = document.createElement('H3');
+  var $topParagraph = document.createElement('P');
+  $topParagraph.classList.add('width-paragraph');
+  $topParagraph.textContent = data.entries[indexValue].Notes;
+  $titleH3.textContent = data.entries[indexValue].Title;
+
+  $rightContainer.appendChild($titleH3);// appends the title onto the text div
+  $rightContainer.appendChild($topParagraph);// appends the p onto the text div
+
+  $divImage.appendChild($img); // appends img to the div for images
+  $entriesSpace.appendChild($divImage);// is appended to the actual div on html
```

Comment:
the only update that is needed here is to move the appending to the DOM outside of the function. This function's job should be to create the DOM tree for the entry and then return it. You want to do the actual appending to the DOM in the loop which is calling this function. Make sure to take a look at the `DOM creation` exercise for a good example of this functionality.